config/samples/node_example_loadtest_with_pre_built_workers.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: e2etest.grpc.io/v1
+kind: LoadTest
+metadata:
+  # Every load test instance must be assigned a unique name on the
+  # cluster. There are ways we can circumvent naming clashes, such
+  # as using namespaces or dynamically assigning names.
+  name: node-example
+
+  # As a custom resource, it behaves like a native kubernetes object.
+  # This means that users can perform CRUD operations through the
+  # Kubernetes API or kubectl. In addition, it means that the user
+  # can set any metadata on it.
+  labels:
+    language: node
+spec:
+  # The user can specify servers to use when running tests. The
+  # initial version only supports 1 server to limit scope. Servers
+  # is an array for future expansion.
+  #
+  # There are many designs and systems to pursue load balancing,
+  # organizing and monitoring a mesh of servers. Therefore, this
+  # will likely be expanded in the future.
+  servers:
+  - language: node
+    run:
+      image: gcr.io/grpc-testing/e2etest/prebuild/node
```

Comment:
I usually test these images in my repo, like gcr.io/grpc-testing/wanlin/prebuild/node instead of the official repo for release. I think @codeblooded know more if this is okay.

Reply:
agree, done